VAMobile/e2e/tests/Navigation.e2e.ts

```diff
@@ -1,3 +1,9 @@
+/*
+Description:
+Detox script that runs dark mode, landscape, and font size accessibility tests
```

Comment:
Does it make sense to add a line describing the difference between running this in the context of the full suite, vs a subset? Or would you expect folks to step through the code to make that leap?

I'd personally add it, since I've fielded that question from our FE before.